R/mod_reportPreview.R

```diff
@@ -172,6 +174,33 @@ reportPreviewServer <- function(id, selected_pkg, maint_metrics, com_metrics,
                 )
               } else "",
               
+              if('Package Dependencies' %in% report_includes()) {
+                tagList(
+                  br(), br(),
+                  hr(),
+                  fluidRow(
+                    column(width = 12,
+                           h5("Package Dependencies",
+                              style = "text-align: center; padding-bottom: 50px;"),
+                             metricGridUI(session$ns('dep_metricGrid'))
+                    )
+                  ),
+                  br(), br(),
+                  fluidRow(
+                    column(width = 8,
```

Comment:
I don't understand why the table is set up to only be 2/3's of the width. I don't think it looks particularly good especially since other tables are full width. We also are not making the table 2/3's width in the downloaded reports.

I guess this also raises the question on why the table is 2/3's width on the dependency tab itself. I believe it was to incorporate the dependency plot, but that plot was scrapped.

Reply:
OK set it to width = 12